src/haz3lweb/www/style.css

```diff
@@ -1532,6 +1532,12 @@ svg.expandable path {
   font-size: 75%;
   top: -10px;
 }
+.DHCode .InvalidFloatOfString{
```

Comment:
as a cleanup can we just have a single class for styling all such run-time errors instead of copying this for each new error